21270.4 <c-7#109> Created logger
21270.4 <c-7#109> appl_initialize was called
21270.4 <c-7#109:9> Logger was updated
21270.4 <c-7#109:9> Started intializing cube lib
21270.4 <c-7#109:9> Finished intializing cube lib with worker, starting to work
21270.5 <c-7#109:9> requestCubes => [47] Buffer size: 0
21270.5 <c-7#109:9> Sent requestCubes signal to root
21270.5 <c-7#109:9> Received 64 cubes from root
21270.5 <c-7#109:9> The main loop continues
21270.5 <c-7#109:9> Started solving a cube
21273.1 <c-7#109:9> Cube failed
21273.1 <c-7#109:9> Added failed cube
21273.1 <c-7#109:9> Started solving a cube
21273.1 <c-7#109:9> Cube failed
21273.1 <c-7#109:9> Added failed cube
21273.1 <c-7#109:9> Started solving a cube
21273.1 <c-7#109:9> Cube failed
21273.1 <c-7#109:9> Added failed cube
21273.1 <c-7#109:9> Started solving a cube
21275 <c-7#109:9> Cube failed
21275 <c-7#109:9> Added failed cube
21275 <c-7#109:9> Started solving a cube
21361.8 <c-7#109:9> appl_interrupt was called
21361.8 <c-7#109:9> Interrupting worker
21361.8 <c-7#109:9> STOP (0.00s since last cb)
21361.8 <c-7#109:9> Solving interrupted
21361.8 <c-7#109:9> The main loop continues
21361.8 <c-7#109:9> appl_interrupt was called
21361.8 <c-7#109:9> appl_withdraw was called
21361.8 <c-7#109:9> Leaving the main loop due to an interruption
21361.8 <c-7#109:9> Started withdraw thread
21361.8 <c-7#109:9> Finished withdrawing lib
21361.8 <c-7#109:9> Enter destructor of CubeWorker
21361.8 <c-7#109:9> Time waiting for messages: 0.002
21361.8 <c-7#109:9> Enter destructor of CubeWorkerInterface
21361.8 <c-7#109:9> Enter destructor of CubeRoot
21361.8 <c-7#109:9> Finished withdraw thread
21424.1 <c-7#109:9> Enter destructor
21424.1 <c-7#109:9> Time working: 91.387
21424.1 <c-7#109:9> Time suspended: 0.000
21424.1 <c-7#109:9> Joined cleanup thread
21424.1 <c-7#109:9> Exit destructor
21424.1 <c-7#109:9> Destructing logger
